test/collection_test.js

```diff
@@ -174,6 +174,55 @@ describe("Collection", () => {
     });
   });
 
+  describe("#_setPlugins", () => {
+    var articles;
+
+    beforeEach(() => articles = testCollection());
+
+    it("should check that first argument is a supported type", () => {
+      try {
+        articles._setPlugins("bogus", {});
```

Comment:
So what would be the expected/desired behavior if someone does:
````js
let a = kinto.collection("articles");
let b = kinto.collection("articles", { remoteTransformers: [ myTransformer ] });
````
Should the second argument be ignored if the collection is already cached? Or should the collections not be cached at all, maybe? (given that there will now be many ways to parameterize them).